src/openfermion/functionals/get_one_norm.py

```diff
@@ -0,0 +1,130 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""
+Function to calculate the 1-Norm of a molecular Hamiltonian after
+fermion-to-qubit transformation from a MolecularData class.
+See https://arxiv.org/abs/2103.14753
+"""
+
+import numpy as np
+
+from openfermion import MolecularData
+
+
+def get_one_norm(mol_or_int, return_constant=True):
```

Comment:
Be explicit with inputs.  Use default None and allow the user to specify.  This partially documents the function.

Reply:
It will also give you freedom to calculate norms for 1-body ops only.  or 2-body only.  Is there a difficulty extending this to arbitrary interactions?

Reply:
Hmmm, this is only valid in the first place for two-body tensors which have real orbital symmetries. I'm not sure how to extend this, as an n-body interaction will not be a molecular system. There is probably a way to write an arbitrary n-body Fermionic Hamiltonian in Majorana representation in terms of the original coefficients though? (I just wouldn't know how)